src/components/BookItem.js

```diff
@@ -1,44 +1,73 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import { useSelector, useDispatch } from 'react-redux';
+import { useDispatch } from 'react-redux';
+import { CircularProgressbar } from 'react-circular-progressbar';
+import 'react-circular-progressbar/dist/styles.css';
 import { removeBook } from '../redux/books/books';
 
-function BookItem() {
-  const bookStore = useSelector((state) => state.booksReducer);
+const BookItem = ({ book }) => {
+  const {
+    title, author, id, category,
+  } = book;
   const dispatch = useDispatch();
 
-  const handleRemove = (id) => {
+  const HandleRemove = () => {
     dispatch(removeBook(id));
   };
-  return (
-    bookStore.map((book) => (
-      <li key={book.id}>
-        <span>
-          {' '}
-          {book.title}
-          {' '}
-        </span>
-        <span>
-          {' '}
-          {book.author}
-          {' '}
-        </span>
 
-        {' '}
-        <button type="button" onClick={() => handleRemove(book.id)}>Remove</button>
-      </li>
-    ))
+  const chapterVal = Math.round(Math.random() * 10);
 
+  return (
+    <li key={id} className="d_flex">
+      <div className="firsCol">
+        <p className="category">{category}</p>
+        <p className="title">{title}</p>
+        <p className="Author" required>
+          {author}
+        </p>
+        <div className="firsCol-btns">
+          <button type="button">Comment</button>
+          <button type="button" onClick={HandleRemove}>
+            Remove
+          </button>
+          <button type="button">Edit</button>
+        </div>
+      </div>
+      <ul className="secondCol d_flex">
+        <li>
+          <div style={{ width: 100, height: 100 }}>
+            <CircularProgressbar value={Math.round((chapterVal / 10) * 100)} />
+          </div>
+        </li>
+        <li>
+          <p className="percentageVal">
+            {Math.round((chapterVal / 10) * 100)}
+            %
+          </p>
+          <p className="completed">completed</p>
+        </li>
+        <li>
+          <div className="lastCol">
+            <p className="currChapter">CURRENT CHAPTER</p>
+            <p id="chapter">
+              Chapter
+              {' '}
+              {chapterVal}
+            </p>
+            <button type="button">UPDATE PROGRESS</button>
+          </div>
```

Comment:
### Your Button
![your-button](https://user-images.githubusercontent.com/93039359/187047379-cb85482d-2ba7-483b-856f-1c5323d17931.png)

### Design Button
![Screenshot from 2022-08-28 01-41-21](https://user-images.githubusercontent.com/93039359/187047384-6367dbf0-0728-4aa7-bbb4-6942ef560395.png)

- [ ] Please make your button look like the design button to make your project look professional :blush:
**Hint: You can add some padding-top and padding-bottom which will do your work :+1:**